src/components/2019/year-in-review/Nav.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import s from 'styled-components'
+import { Link } from 'gatsby'
+
+import { OPEN_SANS_BOLD } from '../../../utils/font'
+
+const Nav = s.div`
+    ${OPEN_SANS_BOLD}  
+    display: flex;
+    max-height: 36px;
+    background-color: #E7525F;
+`
+
+const NavBar = () => {
+  return (
+    <Nav
+      className="navbar navbar-expand-lg navbar-dark sticky-top"
+    >
+      <div className="container">
+        <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar10">
+                <span className="navbar-toggler-icon"></span>
+          </button>
```

Comment:
Is this button even visible? If not, remove the button and simplify html attributes accordingly in this JSX blob

Reply:
it shows up when screen size is small